docs/source/model_design/risk_factors.rst

```diff
@@ -531,9 +531,199 @@ Difference measures relating to the whole population tend to be more useful and
 Population Attributable Fraction (PAF)
 ++++++++++++++++++++++++++++++++++++++
 
+The population attributable fraction is a quantification of the proportion of cases of a given cause that could be eliminated by removing a risk exposure.
+
+..todo::
+	Fill in PAF overview
+
+Bias in PAF Calculation
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The PAF can be calculated using the following formula:
+
+.. math::
+	:label: eq1
+
+	\text{PAF}_{pop}=\frac{p_c(RR-1)}{RR}
+
+In which we define :math:`p_c` to be the proportion of cases that are exposed, 
+and *RR* has been adjusted for confounding and effect modification.
```

Comment:
I wonder if we should refer to RR as RR_adj to indicate that it has been adjusted for confounding? And otherwise RR_crude? I am definitely able to follow without it, but I'm thinking this might help keep it straight.

It does get tricky if the RR is from an RCT or if the RR_adj = RR_crude, though, so feel free to ignore if you think it is less complicated as is :)